### PR TITLE
Updated: Added new pipeline example for the tutorial docs (Issue #11208)

### PR DESCRIPTION
**Pull request by sanchitburkule**
Added hooks and changed whats next position as per request from https://github.com/apache/airflow/pull/16084

### CODE REVIEW

**Comment by Sanchit112** [comment]
@mik-laj @kaxil

**Comment by potiuk** [comment]
I think the whole .csv file was added again ? That was unintentional ?

File: docs/apache-airflow/tutorial.rst [hunk 1]
```diff
@@ -358,25 +358,6 @@ which are used to populate the run schedule with task instances from this dag.
...
-    - Take an in-depth tour of the UI - click all the things!
-    - Keep reading the docs!
-
-      - Review the :doc:`how-to guides<howto/index>`, which include a guide to writing your own operator
-      - Review the :ref:`Command Line Interface Reference<cli>`
-      - Review the :ref:`List of operators <pythonapi:operators>`
-      - Review the :ref:`Macros reference<macros>`
-    - Write your first pipeline!
-
 Lets look at another example; we need to get some data from a file which is hosted online and need to insert into our local database. We also need to look at removing duplicate rows while inserting.
```
Comment (mik-laj) [inline docs/apache-airflow/tutorial.rst:365]: Can you add a new header here to separate two examples?

File: docs/apache-airflow/tutorial.rst [hunk 2]
```diff
@@ -426,17 +407,16 @@ Let's break this down into 3 steps: get data, insert data, merge data:
...
   @task
   def insert_data():
-      engine = create_engine(
-          "postgresql+psycopg2://postgres:password@localhost:5432/postgres"
-      )
+      postgres_hook = PostgresHook(postgres_conn_id="LOCAL")
+      conn = postgres_hook.get_conn()
       df = pd.read_csv("/usr/local/airflow/dags/files/employees.csv")
-      df.to_sql("Employees_temp", con=engine, if_exists="replace", chunksize=1000)
+      df.to_sql("Employees_temp", conn, if_exists="replace", chunksize=1000)
```
Comment (mik-laj) [inline docs/apache-airflow/tutorial.rst]: Are you sure if it works? Based on Pandas documentation, Dataframe supports [`sqlalchemy.engine.(Engine or Connection) or sqlite3.Connection`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_sql.html) only. Here, we have `psycopg2.extensions.connection`?

**Review by potiuk — APPROVED** [review]

File: docs/apache-airflow/tutorial.rst [hunk 3]
```diff
@@ -426,17 +411,18 @@ Let's break this down into 3 steps: get data, insert data, merge data:
           for row in response.text.split("\n"):
               file.write(row)
 
-Here we are passing a``GET`` request to get the data from the URL and save it in ``employees.csv`` file on our Airflow instance.
+Here we are passing a`GET` request to get the data from the URL and save it in `employees.csv` file on our Airflow instance.
```
Comment (ashb) [inline docs/apache-airflow/tutorial.rst]: This is reStructuredText, not markdown, so two backticks is correct for inline code

```suggestion
Here we are passing a ``GET`` request to get the data from the URL and save it in ``employees.csv`` file on our Airflow instance.
```

**Comment by mik-laj** [comment]
Static checks are red. Can you fix it?

**Review by potiuk — APPROVED** [review]

File: docs/apache-airflow/tutorial.rst [hunk 4]
```diff
@@ -1,23 +1,3 @@
...
- .. Unless required by applicable law or agreed to in writing,
-    software distributed under the License is distributed on an
-    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-    KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
-
-
-
-
```
Comment (kaxil) [inline docs/apache-airflow/tutorial.rst]: We need the license @Sanchit112 at the top of the file :)

Comment (sanchitburkule) [reply docs/apache-airflow/tutorial.rst]: ok ill add it.

**Comment by Sanchit112** [comment]
@kaxil can you help me with this? Not able to get this error.

**Comment by potiuk** [comment]
YEAH. YET another random pylint problem. Hopefuly we get rid of it by Monday.

In the meantime just do what I did here: https://github.com/apache/airflow/pull/16674/files#diff-1d96ddfb92ff78838a2d87ff6cdbd5cf445e841fbe78ceea9de732744ffc7728R3653

**Review by kaxil — APPROVED** [review]

**Comment by kaxil** [comment]
Going to ignore random Pylint issues

**Pull request merged**